hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlTestSupport.java

```diff
@@ -104,6 +104,9 @@ public static void createTable(String tableName) throws SQLException {
         createTable(tableName, "id INT PRIMARY KEY", "name VARCHAR(100)");
     }
 
+    protected void createTableWithQuotation(String tableName) throws SQLException{
```

Comment:
Name this `createTable` and rename the method above to `createTableWithoutQuotes`.

It will
- the diff will be much smaller
- quoting the name is the usual case in most of the tests